scripts/script.js

```diff
@@ -0,0 +1,8 @@
+window.onload=function(){
+    document.getElementById("hamburger").addEventListener("click", ()=>{
+        document.getElementById("navigation").classList.toggle("open")
+        document.getElementById("header").classList.toggle("sticky")
```

Comment:
Adding the 'sticky' property to the element causes the page text to jump up because the header element doesn't occupy space in the parent element anymore.

To avoid the jump, you can add a margin-top space to the 'main' when the header is set as sticky.